src/components/app/App.js

```diff
@@ -71,9 +73,32 @@ export default function App({ initialActivities }) {
     </ThemeProvider>
   )
 
+  function handleBookmark(event, id) {
+    event.stopPropagation()
+
+    const activity = activities.find(card => card.id === id)
+
+    const indexActivities = activities.findIndex(card => card.id === id)
```

Comment:
I would just call it `index` or if you want to add the specifity of the activity: `activityIndex`. But really, just `index` would be sufficient enough.

Reply:
Renamed.